src/content/api/module-methods.md

```diff
@@ -91,9 +101,15 @@ import(
 
 T> Note that both options can be combined like so `/* webpackMode: "lazy-once", webpackChunkName: "all-i18n-data" */`. This is parsed as a JSON5 object without curly brackets.
 
+`webpackInclude`: A regular expression that will be matched against during import resolution and point onto modules that __should be bundled__.
```

Comment:
this documentation could be a little more clear. 

**only** modules that match `webpackInclude` will be bundled.
any module that matches `webpackExclude` **will not** be bundled.

it may also be helpful (but not necessary) to use a diagram such as this:
![image](https://user-images.githubusercontent.com/7049913/39836373-662be4ca-5390-11e8-8224-ae7db63779c6.png)

Reply:
To be honest **i like the diagram**, but doesnt feel right to add an image there as maintaining images as features change is a pain in the docs :)